lib/client/api_matrix.es6.js

```diff
@@ -324,64 +325,64 @@ foam.CLASS({
       args: [
         {
-          name: 'browserKeys',
+          name: 'releaseKeys',
           typeName: 'StringArray',
-          documentation: `A string array contains valid browser keys.`,
+          documentation: `A string array contains valid release keys.`,
         },
       ],
       returns: {
         typeName: 'JSON',
         documentation: `A JSON of the form:
-          {
-            browserKey:
             {
-              total: An integer, the total number of APIs belongs to
-                  this browser.
-              proprietary: An integer, number of APIs that only exists
-                  in this browser.
-              fallBehind: An integer, number of APIs that exists in all
-                  other selected browsers but not this browser.
-            }
-          }.`,
+              <releaseKey>:
+              {
+                total: An integer, the total number of APIs belongs to
+                    this release.
+                proprietary: An integer, number of APIs that only exists
+                    in this release.
+                fallBehind: An integer, number of APIs that exists in all
+                    other selected releases but not this release.
+              }
+            }.`,
       },
-      code: function(browserKeys) {
-        return this.browserApiDAO
+      code: function(releaseKeys) {
+        return this.releaseApiDAO
           .where(this.mlang.IN(
-            this.BrowserWebInterfaceJunction.SOURCE_ID, browserKeys))
+            this.ReleaseWebInterfaceJunction.SOURCE_ID, releaseKeys))
           .select(this.mlang.GROUP_BY(
-            this.BrowserWebInterfaceJunction.TARGET_ID,
+            this.ReleaseWebInterfaceJunction.TARGET_ID,
             this.ArraySink.create()))
           .then((groups) => {
             let result = {};
-            let totalBrowsers = browserKeys.length;
+            let totalReleases = releaseKeys.length;
             let interfaceKeys = groups.groupKeys;
-            for (let i = 0; i < browserKeys.length; i++) {
-              result[browserKeys[i]] = {
+            for (let i = 0; i < releaseKeys.length; i++) {
+              result[releaseKeys[i]] = {
                 total: 0,
                 proprietary: 0,
                 fallBehind: 0,
               };
             }
             for (let i = 0; i < interfaceKeys.length; i++) {
               let interfaceKey = interfaceKeys[i];
-              let availableBrowsers = {};
-              let browsers = groups.groups[interfaceKey].a;
-              for (let j = 0; j < browsers.length; j++) {
-                result[browsers[j].sourceId].total++;
-                availableBrowsers[browsers[j].sourceId] = true;
-                if (browsers.length === 1) {
-                  result[browsers[j].sourceId].proprietary++;
+              let availableReleases = {};
+              let releases = groups.groups[interfaceKey].a;
+              for (let j = 0; j < releases.length; j++) {
```

Comment:
General comment that's completely unrelated to this PR, since I think I've seen it all over the place.

All of these for loops that use the index only to look up a value in an array should probably be foreach loops. I think a comment I made about being consistent in which one was used when it didn't matter may have contributed to the current situation.